tests/nose2.cfg

```diff
@@ -0,0 +1,27 @@
+[unittest]
+plugins = nose2.plugins.logcapture
+          nose2.plugins.junitxml
+          nose2.plugins.mp
+
+[test-result]
+always-on = True
+descriptions = True
+
+[junit-xml]
+always-on = True
+test_fullname = True
+
+[log-capture]
+always-on = True
+clear-handlers = True
+log-level = DEBUG
+
+[multiprocess]
+always-on = True
+test-run-timeout = 600
+
+[coverage]
+always-on = True
+coverage-report = term \
+                  xml
```

Comment:
The coverage report is missing from the ci logs ? Probably due to these lines, why did you added them ?

Reply:
To have both, a report in terminal and also a more detailed report as .xml file.
I'll check why the report is missing in ci, that's strange

Reply:
Still not in the CodeBuild output, but it appears in the dedicated report group now at least.
Digging further

Reply:
Now we have the coverage report in CodeBuilds terminal output (but therefore no longer in an xml file). For me its fine like that. OK for you?